core/src/test/scala/integration/kafka/server/GssapiAuthenticationTest.scala

```diff
@@ -109,7 +110,8 @@ class GssapiAuthenticationTest extends IntegrationTestHarness with SaslSetup {
    * are able to connect after the second re-login. Verifies that logout is performed only once
    * since duplicate logouts without successful login results in NPE from Java 9 onwards.
    */
-  @Test
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
```

Comment:
There is no input parameter to `testLoginFailure()` -- why change this to being a parametrized test? (Same below.)

Reply:
This was supposed to take a parameter. I'll fix it (and the one below). See the explanation above.